package/upgrade/migrations/managed_charts/v1.1.0.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -ex
+
+CHART_NAME=$1
+CHART_MANIFEST=$2
+
+patch_grafana_resources()
+{
+  # Increase grafana pod limit and request (https://github.com/harvester/harvester-installer/pull/287)
+  yq e '.spec.values.grafana.resources = {"limits": {"cpu": "200m", "memory": "500Mi"}, "requests": {"cpu": "100m", "memory": "200Mi"}}' $CHART_MANIFEST -i
+
+  # Increase grafana PVC size to 2Gi (https://github.com/harvester/harvester-installer/pull/327)
+  yq e '.spec.values.grafana.persistence.size = "2Gi"' $CHART_MANIFEST -i
```

Comment:
only patching the PVC size values without actually changing the real volume size is not valuable and may make it ambiguous for the user, would prefer to leave this unchanged and use both docs and release notes to provide a manual workaround solution, thanks.